hms-app/src/pages/IdeaPortal.tsx

```diff
@@ -161,12 +161,10 @@ function IdeaPortal() {
     return item.title?.toLowerCase().includes(searchTerm.toLowerCase())
   })
 
-  const data = hackathonList.hackathons.map(
-    (hackathon, index) => hackathon.title
-  )
+  const data = hackathonList.users.map((hackathon, index) => hackathon.title)
```

Comment:
`data` isn't a great name for this.  Maybe `hackathonTitles`? but doing this also feels not great because you lose the linking between ID and title :thinking: